index.md

```diff
@@ -1,49 +1,41 @@
 ---
-layout: default
+layout: home
 title: Home
 section: Overview
 ---
+<p class='alert-big'>Origami is a group of services, modules and tools to make building websites at the FT simpler and faster.</p>
 
-<p class="alert-big">Origami is a set of standards, conventions and tools for building websites at the FT.</p>
+<h2> <i class="fa fa-cubes"></i> Origami modules and services</h2>
 
-<p class="alert-big">Origami aspires to create a <strong>unified style and experience for FT websites</strong>, and <strong>make web development at the FT faster</strong>.</p>
+At the heart of Origami are its services and modules.
 
-This standard is equally for component developers and those who want to build websites using existing components.  These two roles will be distinguished in this standard as **component developer** and **product developer**.
+Origami's modules are bits of HTML, CSS and JavaScript to include in your page. Some of them, like [o-gallery](http://registry.origami.ft.com/components/o-gallery), are fully formed ready to use components. Some of them, like [o-colors](http://registry.origami.ft.com/components/o-colors) or [o-tracking](http://registry.origami.ft.com/components/o-tracking) are tools to incorporate into your own pages and build with.
```

Comment:
Not sure I like this description of tracking and colors.  gallery is also a component to incorporate into your page and build with.  Perhaps you're wanting to say something like "Some of them, like gallery, come with a fully formed look and feel.  Some of them, like colors or tracking provide a foundation for building your own interfaces, or provide useful services without any interface at all"

Also: repetition of 'some of them' - 'others' would be better on the second instance.  And use of 'like' vs 'such as' or 'for example' is dodgy ground. Rob used to tell me off quite frequently for overuse of like.  In this case I think like is correct because 'some of them' is too broad a term so it needs comparison to work.  It's components that are comparable to gallery, not a set of components that includes gallery.